.travis.yml

```diff
@@ -49,6 +49,9 @@ before_install:
 install:
   - travis_retry composer install --no-interaction --ignore-platform-reqs
 
+before_script:
+  echo 'dmbstring.func_overload=7' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
```

Comment:
Actually... why do this? With the new test cases, if somebody reverts to the non-`mb_*()` equivalents, adding this setting will essentially overload them with the `mb_*()` equivalent, and they'd continue to pass. If this is *not* set, then the tests will catch the regression... Or am I missing something?